modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/config/ApplicationConfiguration.java

```diff
@@ -33,15 +34,17 @@
 @EnableScheduling
 public class ApplicationConfiguration {
     /**
+     * @param messages Messages accessor.
+     *
      * @return Service for encoding user passwords.
      */
     @Bean
-    public PasswordEncoder passwordEncoder() {
+    public PasswordEncoder passwordEncoder(MessageSourceAccessor messages) {
```

Comment:
Why not create MessageSource like here: org.springframework.security.core.SpringSecurityMessageSource